specification/maps/data-plane/Common/preview/1.0/common.json

```diff
@@ -1,712 +1,712 @@
-    "title": "Azure Maps Common Types",
-    "version": "1.0"
-  },
-  "x-ms-parameterized-host": {
-    "hostTemplate": "{geography}.atlas.microsoft.com",
-    "parameters": [
-      {
-        "$ref": "#/parameters/GeographicResourceLocation"
-      }
-    ]
-  },
-  "schemes": [
-    "https"
-  ],
-  "consumes": [],
-  "produces": [
-    "application/json"
-  ],
-  "paths": {},
-  "definitions": {
-    "LongRunningOperationResult": {
-      "description": "The response model for a Long-Running Operations API.",
-      "type": "object",
-      "properties": {
-        "operationId": {
-          "description": "The Id for this long-running operation.",
-          "type": "string"
-        },
-        "status": {
-          "description": "The status state of the request.",
-          "type": "string",
-          "enum": [
-            "NotStarted",
-            "Running",
-            "Failed",
-            "Succeeded"
-          ],
-          "x-ms-enum": {
-            "name": "LroStatus",
-            "modelAsString": true,
-            "values": [
-              {
-                "value": "NotStarted",
-                "description": "The request has not started processing yet."
-              },
-              {
-                "value": "Running",
-                "description": "The request has started processing."
-              },
-              {
-                "value": "Failed",
-                "description": "The request has one or more failures."
-              },
-              {
-                "value": "Succeeded",
-                "description": "The request has successfully completed."
-              }
-            ]
-          },
-          "readOnly": true
-        },
-        "created": {
-          "description": "The created timestamp.",
-          "type": "string",
-          "format": "date-time",
-          "readOnly": true
-        },
-        "error": {
-          "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorDetail"
-        },
-        "warning": {
-          "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorDetail"
+  {
+    "swagger": "2.0",
+    "info": {
+      "title": "Azure Maps Common Types",
+      "version": "1.0"
+    },
+    "x-ms-parameterized-host": {
+      "hostTemplate": "{geography}.atlas.microsoft.com",
+      "parameters": [
+        {
+          "$ref": "#/parameters/GeographicResourceLocation"
         }
-      }
+      ]
     },
-    "BatchRequestBody": {
-      "description": "This type represents the request body for the Batch service.",
-      "type": "object",
-      "properties": {
-        "batchItems": {
-          "description": "The list of queries to process.",
-          "type": "array",
-          "items": {
-            "description": "Batch Query object",
+    "schemes": [
+      "https"
+    ],
+    "consumes": [],
+    "produces": [
+      "application/json"
+    ],
+    "paths": {},
+    "definitions": {
```

Comment:
The Error models that are repeated in nearly every service swagger should be consolidated and localized here to eliminate duplication. Same for security definitions, if possible.